nats-box.yaml

```diff
@@ -0,0 +1,45 @@
+package:
+  name: nats-box
+  version: 0.17.0
+  epoch: 0
+  description: NATS utilities toolbox.
+  dependencies:
+    runtime:
```

Comment:
Needs additional runtime deps: https://github.com/nats-io/nats-box/blob/6cdf3ae5d1e4499137bc650ea1b8d8fc3adb9197/Dockerfile#L20-L21

Reply:
Also see: https://github.com/nats-io/nats-box/blob/6cdf3ae5d1e4499137bc650ea1b8d8fc3adb9197/profile.sh#L3

Reply:
Will we have a compat package for `/usr/local/bin/nats-top`?

Reply:
Yes, already added it in nats-box image addition PR.